### ISSUE TITLE
[BUG] type declaration for eslint flat config

### ISSUE THREAD

**Issue by TheElegantCoding** [francoismassart/eslint-plugin-tailwindcss#371]
**Describe the bug**
The package don't have a type declaration file of the options and exports members

**To Reproduce**
Add the configuration like bellow and add in tsconfig.json `"noImplicitAny": true`

```ts
import tailwind from 'eslint-plugin-tailwindcss';

import type { Linter } from 'eslint';

const Tailwind: Linter.Config =
{
  languageOptions:
  {
    parserOptions:
    {
      ecmaFeatures:
      {
        jsx: true
      }
    }
  },
  name: 'tailwind',
  plugins: tailwind,
  rules:
  {

  }
};

export { Tailwind };
```

**Expected behavior**
This package should support typescript

**Screenshots**
![image](https://github.com/user-attachments/assets/7edd4759-2cce-4750-a57a-44a8ea176af5)

**Environment (please complete the following information):**
 - OS: windows 10
 - Softwares + version used:
     -  VSCode 1.94.2

**Comment by kachkaev** [comment]
There exists [@types/eslint-plugin-tailwindcss](https://www.npmjs.com/package/@types/eslint-plugin-tailwindcss) which you can install in the meantime.

**Comment by TheElegantCoding** [comment]
This work perfect, didnt know that `[@types/eslint-plugin-tailwindcss](https://www.npmjs.com/package/@types/eslint-plugin-tailwindcss)` exists, thanks!

**Comment by kachkaev** [comment]
The issue you have created is valid, let’s keep it open for now. While `@types/eslint-plugin-tailwindcss` package does the job, it’d be more convenient to use types directly from `eslint-plugin-tailwindcss`. One fewer dependency to install and update!

**Comment by OreQr** [comment]
Why not just add [index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/eslint-plugin-tailwindcss/index.d.ts) directly to `eslint-plugin-tailwindcss`?

**Comment by leetcow** [comment]
this is a blocker for us to add this package as a dependency

**Comment by francoismassart** [comment]
I believe this is available in v4
